src/alarms/notifications/App.js

```diff
@@ -13,6 +13,20 @@ import { FormattedMessage } from "react-intl";
 import { withRouter } from "react-router-dom";
 import classNames from "classnames";
 
+async function simpleAlarmNotificationFetch(url, body) {
```

Comment:
Dit is wel goed, alleen heeft deze functie niets met alarmen te maken zo dus dat kan uit de naam. En de variabele hoeft geen updatedAlarm te heten. Ik zou hem gewoon jsonFetch noemen en url, body (en ook method) parameters maken. En dan niet in alarms/notifications/App maar in een centrale util.

Reply:
okey nieuwe poging ;)